site/sfguides/src/deploying_custom_models_to_snowpark_model_registry/deploying_custom_models_to_snowpark_model_registry.md

```diff
@@ -0,0 +1,168 @@
+author: Mats Stellwall
+id: deploying_custom_models_to_snowpark_model_registry
+summary: Through this quickstart guide, you will learn how to deploy a custom model to the Snowpark Model Registry.
+<!--- Categories below should be hyphenated, i.e., Getting-Started. Do not leave blank. Visit site for available categories. -->
```

Comment:
@sfc-gh-mstellwall Please delete this comment and don't leave a blank line between the attritbutes afterwards.